packages/swapper/src/swappers/JupiterSwapper/models/AccountMeta.ts

```diff
@@ -0,0 +1,84 @@
+// @ts-nocheck
+/* tslint:disable */
+/* eslint-disable */
```

Comment:
@woodenfurniture we already does this in multiple places, those types are generated OPEN API types, I imported only the bits we consume to avoid spamming our repository with unused bits as we did for Chainflip already